test/SerializerTest.php

```diff
@@ -34,7 +35,7 @@ public function testGetDefaultAdapterPluginManager()
 
     public function testChangeAdapterPluginManager()
     {
-        $newPluginManager = new AdapterPluginManager();
+        $newPluginManager = new AdapterPluginManager(new ServiceManager);
```

Comment:
Instead of using a concrete `ServiceManager` instance, you could also just mock the `ContainerInterface`.